openshift-ci/Dockerfile.tools

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/fedora/fedora:31-x86_64
+FROM quay.io/fedora/fedora:33-x86_64
```

Comment:
y not 34 which is the latest?
and contemplating whether we might want something more rigid and LTS (ie not fedora at all)

Reply:
i will update to 34.
maybe we can use centos? @SchSeba what do you think?

Reply:
centos moved to be a stream os
it is not suitable for CI
as you are getting something unpredictable :-(

Reply:
I am open to everything you prefer this is just a CI container

Reply:
debian ? :-D